package.json

```diff
@@ -1,42 +1,34 @@
 {
-  "name": "@mondago/strapi-provider-email-msgraph",
-  "version": "1.0.1",
-  "description": "Microsoft Graph email provider plugin for Strapi 3",
-  "main": "./lib",
+  "name": "strapi-provider-email-msgraph",
+  "version": "1.0.0",
+  "description": "Emails via Microsoft Graph",
+  "main": "./dist",
+  "types": "./dist/index.d.ts",
   "directories": {
-    "lib": "lib"
-  },
-  "dependencies": {
-    "@azure/identity": "^3.0.0",
-    "@microsoft/microsoft-graph-client": "^3.0.2",
-    "isomorphic-fetch": "^3.0.0"
+    "dist": "./dist"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "format:check": "prettier --check src/**/*.{ts,tsx}",
+    "format": "prettier --write src/**/*.{ts,tsx}",
```

Comment:
Since we've got Prettier included it might be worth adding our config

Reply:
Good spot, done

Reply:
You need to also add `"prettier": "@mondago/prettier-config"` to package.json